docs/companion.md

```diff
@@ -640,6 +641,15 @@ enabled, it will lead to _faster uploads_ because companion will start uploading
 at the same time as downloading using `stream.pipe`. If `false`, files will be
 fully downloaded first, then uploaded. Defaults to `true`.
 
+#### `streamingUploadSizeless` `COMPANION_STREAMING_UPLOAD_SIZELESS`
```

Comment:
What happens if it's not enabled? Hard crash? Silent ignore?

Why would we put it behind on option and not simply also allow it and return the error from the tus server if it doesn't support it (which is highly unlikely, all tus servers I know support it)?